contracts/interfaces/Accounts.sol

```diff
@@ -18,4 +18,8 @@ interface IAccount {
         bytes memory data,
         uint8 operation
     ) external returns (bool success, bytes memory returnData);
+
+    function checkSignatures(bytes32 dataHash, bytes memory data, bytes memory signatures) external view;
```

Comment:
Might be worth documenting somewhere (if it isn't already the case), that this method must revert on an invalid signature.

Reply:
Yes, we will update in specs.